alignak/acknowledge.py

```diff
@@ -83,7 +85,7 @@ class Acknowledge(AlignakObject):  # pylint: disable=R0903
     # of the Alignak process. If not, the comment will be deleted the
     # next time Alignak restarts.
 
-    def serialize(self):
+    def serialize(self, filtered_fields=None):
```

Comment:
Really necessary to document a property inherited from a base class? Seems really unuseful to me...